### DISCUSSION TITLE
How to update fusion 360?

### DISCUSSION THREAD

**Discussion by rcludwick** [cryinkfly/Autodesk-Fusion-360-for-Linux#269] [Q&A]
When I run fusion 360 through the launcher, a dialog window pops trying to connect to a server.  

If I hit ok, it asks me if I want to update fusion 360.  

It then proceed to tell me the update has been skipped.  

I would rather not have to go through the process of reinstalling F360 for linux all over again.

**Comment by Rafgass** [comment]
Same issue!

**Comment by overbeck** [comment]
Same here.

**Comment by S377** [comment]
Hi,
Same here !
Using htop, I could see that the installer (for the update I suppose) is effectively downloaded, and then copied to 

> /home/\<user\>/.fusion360/wineprefixes/default/drive_c/users/\<user\>/Downloads

But then, it's Fusion360 that is launched and not its update !

And if I launch it manually with wine, it seems to freeze on the Fusion install's splash screen...
